tests/testChain.cpp

```diff
@@ -0,0 +1,80 @@
+/* ----------------------------------------------------------------------------
+ * GTDynamics Copyright 2020, Georgia Tech Research Corporation,
+ * Atlanta, Georgia 30332-0415
+ * All Rights Reserved
+ * See LICENSE for the license information
+ * -------------------------------------------------------------------------- */
+
+/**
+ * @file  testChain.cpp
+ * @brief Test Chain class.
+ * @author: Dan Barladeanu, Frank Dellaert
+ */
+
+#include <CppUnitLite/TestHarness.h>
+
+#include <gtsam/geometry/Pose3.h>
+#include <gtsam/base/Matrix.h>
+
+#include "gtdynamics/dynamics/Chain.h"
+
+using namespace gtdynamics;
+using gtsam::Pose3;
+using gtsam::Rot3;
+using gtsam::Point3;
+using gtsam::Matrix;
+using gtsam::assert_equal;
+
+TEST(threeLinks, All) {
+  
+  Pose3 sMb = Pose3(Rot3(), Point3(5, 0, 0));
```

Comment:
Make nice blocks, each with a `// comment` to give some guidance as to what's happening.

Reply:
done